tests/tasks/teardown/kit.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Ellis and I discussed over this in the past and came to this consensus, we are keeping things simple where we can if we can convey it in file name itself instead of an additional directory.